evap/evaluation/templates/startpage_button.html

```diff
@@ -0,0 +1,13 @@
+{% if user.startpage == page %}
+    <div data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'This is your startpage' %}">
+        <button type="button" class="btn btn-sm btn-light" disabled><span class="fas fa-house"></span></button>
+    </div>
+{% else %}
+    <form class="d-flex" method="post" action="{% url 'evaluation:set_startpage' %}">
+        {% csrf_token %}
+        <input name="page" type="hidden" value="{{ page }}">
+        <button id="makeStartPageButton" type="submit" class="btn btn-sm btn-light" onclick="setSpinnerIcon('span-set-startpage')" data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'Make this my startpage' %}">
+            <span id="span-set-startpage" class="fas fa-house"></span>
+        </button>
+    </form>
+{% endif %}
```

Comment:
How do you feel about the following, to make sure that the buttons look the same in both cases (I don't like the duplicated html)?

```html
<form class="d-flex" method="post" action="{% url 'evaluation:set_startpage' %}" onsubmit="setSpinnerIcon('span-set-startpage')">
    {% csrf_token %}
    <input name="page" type="hidden" value="{{ page }}">
    <span
        data-bs-toggle="tooltip"
        data-bs-placement="left"
        {% if user.startpage == page %}
            title="{% trans 'This is your startpage' %}"
        {% else %}
            title="{% trans 'Make this my startpage' %}"
        {% endif %}
    >
        <button
            id="makeStartPageButton"
            type="submit"
            class="btn btn-sm btn-light"
            {% if user.startpage == page %}disabled{% endif %}
        >
            <span id="span-set-startpage" class="fas fa-house"></span>
        </button>
    </span>
</form>
```

Reply:
i prefer not to have a form at all if not necessary. i see your point, but find the approach with two separate definitions more readable. in theory, the icon on the disabled button could also be different (e.g., a person inside the house) - i don't think of it as "the same button".

Reply:
I am not sure I agree, but we can do it your way :)